commands/admin/unlock.py

```
@@ -3,66 +3,32 @@
 import discord
 
 import utility
-from api import clear_channel_overwrites, get_channel_overwrites
 from localizer import tanjunLocalizer
+from utility import CommandInfo
```

Flake8 (7.3.0)

[error] 'utility.CommandInfo' imported but unused

(F401)